connector_salesforce/salesforce_contact/importer.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Nicolas Bessi
+#    Copyright 2014 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+import logging
+from openerp.addons.connector.exception import MappingError
+from openerp.addons.connector.unit.mapper import mapping, only_create
+from ..backend import salesforce_backend
+from ..unit.importer_synchronizer import (SalesforceDelayedBatchSynchronizer,
+                                          SalesforceDirectBatchSynchronizer,
+                                          SalesforceImportSynchronizer,
+                                          import_record)
+from ..unit.rest_api_adapter import SalesforceRestAdapter
+from ..unit.mapper import AddressMapper
+_logger = logging.getLogger('salesforce_connector_contact_import')
```

Comment:
Why not `__name__`? You would have the full path, so `connector_salesforce.salesforce_contact.importer` which seems clean to me.